recipes/myerson/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "myerson" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/myerson-{{ version }}.tar.gz
+  sha256: 5f1b31534132a6e2794d73a6e43c8cefd08cc56f55066776c4ba54a8256ec508
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >=3.8
+    - tqdm
+    # - pytorch >=2.0.1,<3
```

Comment:
Why is thia commented out?

Reply:
The PyTorch dependency is optional and apparently there is no PyTorch on conda-forge for Windows.

Reply:
That's OK. CI-on-windows failure is OK here on staged-recipes for noarch packages.

Would you like to add it as a `run_constraints` instead?